src/js/device/DeviceCommands.js

```diff
@@ -258,6 +258,39 @@ export default class DeviceCommands {
         return this.typedCall('StellarSignTx', 'StellarSignedTx', transaction);
     }
 
+    async liskGetAddress(address_n: Array<number>, showOnTrezor: boolean): Promise<trezor.LiskAddress> {
+        const response: MessageResponse<trezor.LiskAddress> = await this.typedCall('LiskGetAddress', 'LiskAddress', {
+            address_n,
+            show_display: !!showOnTrezor,
+        });
+        return response.message;
+    }
+
+    async liskGetPublicKey(address_n: Array<number>, showOnTrezor: boolean): Promise<trezor.LiskPublicKey> {
+        const response: MessageResponse<trezor.LiskPublicKey> = await this.typedCall('LiskGetPublicKey', 'LiskPublicKey', {
+            address_n,
+            show_display: !!showOnTrezor,
+        });
+        return response.message;
+    }
+
+    async liskSignMessage(address_n: Array<number>, message: string): Promise<trezor.LiskMessageSignature> {
+        const response: MessageResponse<trezor.LiskMessageSignature> = await this.typedCall('LiskSignMessage', 'LiskMessageSignature', {
+            address_n,
+            message,
+        });
+        return response.message;
+    }
+
+    async liskVerifyMessage(public_key: string, signature: string, message: string): Promise<trezor.Success> {
+        const response: MessageResponse<trezor.Success> = await this.typedCall('LiskVerifyMessage', 'Success', {
+            public_key,
+            signature,
+            message,
+        });
+        return response.message;
+    }
+
```

Comment:
`liskSignTransaction` method is missing here

Reply:
it doesn't have to be there since this message is called straight from ./js/core/methods/LiskSignTransaction.js line 60.

"signTx" messages for other coins (stellar and nem) will also be (re)moved from this file because their logic is to complex to have it there

Reply:
@szymonlesisz got it. thanks for the clarification.